cmd/btfs/autoupdate.go

```diff
@@ -167,13 +167,19 @@ func update(url, hval string) {
 		}
 
 		// Compare version.
-		flg, err := versionCompare(latestConfig.Version, version)
+		upgradeFlg, err := versionCompare(latestConfig.Version, version)
 		if err != nil {
 			log.Errorf("Version compare error, reasons: [%v]", err)
 			continue
 		}
 
-		if flg <= 0 {
+		if upgradeFlg == 0 {
```

Comment:
There is a small confusion here. If two versions are the same it will also return 0. So people will up-to-date version will see "BTFS will not automatically update" instead of "BTFS is up-to-date".

Suggest doing something like this:

```
const (
    UPGRADE_FLAG_LATEST = iota
    UPGRADE_FLAG_OUT_OF_DATE
    UPGRADE_FLAG_ERR
)
```
And return the constant clearly in `versionCompare`.